assignments/objects.js

```diff
@@ -19,19 +19,62 @@ const example = {
 
 // Write your intern objects here:
 
+const mitzi = {
+  "id": 01,
+  "name": "Mitzi,F",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+const kennan = {
+  "id": 02,
+  "name": "Kennan,M",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak": "Hello, my name is Kennan!"
+}
+
+const keven = {
+  "id": 03,
+  "name": "Keven,M",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const gannie = {
+  "id": 04,
+  "name": "Gannie,M",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const antonietta = {
+  "id": 05,
+  "name": "Antonietta,F",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+  // "multiplyNums": function multiplyNums() {
+  // return a * b;
+}
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(mitzi.name);
 
 // Kennan's ID
 
+console.log(kennan.id);
+
 // Keven's email
+console.log(keven.email);
 
 // Gannie's name
+console.log(gannie.name);
 
 // Antonietta's Gender
+console.log(antonietta.gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
```

Comment:
to give a method to an object you can just make a new key (without quotes) and give it a value of a function. like `speak: function() {console.log("yo I'm Kennan"}` and then call it with `kennan.speak()`